public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,103 @@
+# dvc.api.get_url()
+
+Returns the URL to the storage location of a data file or directory tracked by
+DVC.
+
+## Definition
+
+### Signature
+
+```py
+get_url(path, repo=None, rev=None, remote=None)
+```
+
+### Types
+
+All **parameter** types as well as the **return** type are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+Raises `UrlNotDvcRepoError` if `repo` is not a <abbr>DVC repository</abbr>.
```

Comment:
also, it strange to see exceptions as part of the Types section

Reply:
Exceptions are types technically I think? But sure, I can move this around. Maybe remove **Types** header and just make it a one line paragraph like:

``**Types:** `param1` is `int`. `param2` is [string]``

And then, similarly:

`**Exceptions**: No exceptions raised`

WDYT?

Reply:
p.s. the exceptions part could even be moved to the short def. or to the longer description later, like in stdlib docs e.g.:

![image](https://user-images.githubusercontent.com/1477535/75298429-15f36500-57f8-11ea-8568-3e05ff46b12c.png)

Also notice that they don't even mention the types at all in there. I guess it's a little low-level and can be left for docstrings only... WDYT?

Reply:
I still think that exception should not be part of the Types section. How to change this - make part of the signature (since it's signature it means it should be really really short, may be w/o any descriptions) + in the description section explain them

Reply:
The only 2 exceptions I know can be thrown by all 3 functions are `UrlNotDvcRepoError` or `NoRemoteError`, which are linked more to the parameters (explained in that section already) so I just removed them from the earlier sections for now.